config/routes.rb

```diff
@@ -239,6 +239,12 @@
     as:          :repository_tree,
     constraints: { path: /.*/ }
 
+  delete ':repository_id/*path',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.